### PR TITLE
[docs][yba] Back up YBA using yba-ctl

### LINKED ISSUE #19487: [Docs] [YBA-Installer] Document backup process for YBA-installer
### Description

https://docs.yugabyte.com/stable/yugabyte-platform/administer-yugabyte-platform/back-up-restore-yp/ we need to add a section specific to YBA-installer that says they need to use yba-ctl createBackup 

```
[ec2-user@ip-10-9-133-223 ~]$ sudo yba-ctl createBackup --help

    The createBackup command executes our yb_platform_backup.sh that creates a backup of your
    YugabyteDB Anywhere instance. Executing this command requires that you specify the
    outputPath where you want the backup .tar.gz file to be stored as the first argument to
    createBackup.

Usage:
  yba-ctl createBackup outputPath [flags]

Flags:
      --data_dir string      data directory to be backed up (default "/opt/yugabyte")
      --exclude_prometheus   exclude prometheus metric data from backup (default: false)
      --exclude_releases     exclude YBDB releases from backup (default: false)
  -h, --help                 help for createBackup
      --skip_restart         don't restart processes during execution (default: false)
      --verbose              verbose output of script (default: false)

Global Flags:
  -f, --force              Run in non-interactive mode. All user confirmations are skipped.
      --log_level string   log level for this command. Levels: panic, fatal, error, warn, info, debug, trace. (default "info")
```

And for restore

```
[ec2-user@ip-10-9-133-223 ~]$ sudo yba-ctl restoreBackup --help

    The restoreBackup command executes our yb_platform_backup.sh that restores from a previously
		taken backup of your YugabyteDB Anywhere instance. Executing this command requires that you
		specify the inputPath to the backup .tar.gz file as the only argument to restoreBackup.

Usage:
  yba-ctl restoreBackup inputPath [flags]

Flags:
      --destination string   where to un-tar the backup (default "/opt/yugabyte")
  -h, --help                 help for restoreBackup
      --migration            restoring from a Replicated or Yugabundle installation (default: false)
      --skip_dbdrop          skip dropping the yugaware database before a migration restore (default: false)
      --skip_restart         don't restart processes during execution (default: false)
      --use_system_pg        use system path's pg_restore as opposed to installed binary (default: false)
      --verbose              verbose output of script (default: false)
      --yugabundle           WARNING: yugabundle flag is deprecated.
                             Please use migration instead to migrate from yugabundle to YBA-installer. (default: false)

Global Flags:
  -f, --force              Run in non-interactive mode. All user confirmations are skipped.
      --log_level string   log level for this command. Levels: panic, fatal, error, warn, info, debug, trace. (default "info")
```

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #19487 
Build preview: https://deploy-preview-19526--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <a href="../back-up-restore-k8s/" class="nav-link">
+      <i class="fa-regular fa-dharmachakra" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+</ul>
+
+YugabyteDB Anywhere installations include configuration settings, certificates and keys, and other components required for creating and managing YugabyteDB universes.
+
+You can use the YBA Installer [yba-ctl](../../install-yugabyte-platform/install-software/installer/#download-yba-installer) script which internally executes the `yb_platform_backup.sh` script to back up an existing YugabyteDB Anywhere server and restore it, when needed, for disaster recovery or migrating to a new server.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:38]:
```suggestion
If you installed YBA using [YBA installer](../../install-yugabyte-platform/install-software/installer/), use the [yba-ctl](../../install-yugabyte-platform/install-software/installer/#download-yba-installer) CLI to back up and restore your YBA installation. The CLI executes the `yb_platform_backup.sh` script to back up an existing YugabyteDB Anywhere server and restore it, when needed, for disaster recovery or migrating to a new server.
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+{{< note title="Note" >}}
+
+You cannot back up and restore Prometheus metrics data.
+
+{{< /note >}}
+
+## Back up YBA if installed using YBA Installer
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:46]:
```suggestion
## Back up YBA
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    ./yba-ctl.sh createBackup <output_path> [flags]
+    ```
+
+    The `createBackup` command executes the `yb_platform_backup.sh` script to create a backup of your YugabyteDB Anywhere instance. Executing this command requires that you specify the `output_path` where you want the backup `.tar.gz` file to be stored as the first argument to `createBackup`.
+
+1. Verify that the backup `.tar.gz` file, with the correct timestamp, is in the specified output directory.
+
+1. Upload the backup file to your preferred storage location and delete it from the local disk.
+
+## Restore YBA if installed using YBA Installer
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:62]:
```suggestion
## Restore YBA
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+To restore the YugabyteDB Anywhere content from your saved backup, perform the following:
+
+1. Copy the backup `.tar` file from your storage location.
+
+1. Run the `yba-ctl` script using the `restoreBackup` command:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:68]:
```suggestion
1. Run the `restoreBackup` command as follows:
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+    ```sh
+    yba-ctl restoreBackup <input_path> [flags]
+    ```
+
+    The `restoreBackup` command executes the `yb_platform_backup.sh` to restore from a previously taken backup of your YugabyteDB Anywhere instance. Executing this command requires that you specify the `input_path` to the backup `.tar.gz` file as the only argument to `restoreBackup`.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:74]:
```suggestion
    The `restoreBackup` command executes the `yb_platform_backup.sh` script to restore from a previously taken backup of your YugabyteDB Anywhere instance. Specify the `input_path` to the backup `.tar.gz` file as the only argument.
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+You cannot back up and restore Prometheus metrics data.
+
+{{< /note >}}
+
+## Back up YBA if installed using YBA Installer
+
+If you installed YugabyteDB Anywhere using [YBA installer](../../install-yugabyte-platform/install-software/installer/), perform the following steps:
+
+1. Run the `yba-ctl` script using the `createbackup` command, as follows:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:50]:
```suggestion
1. Run the `createbackup` command, as follows:
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+    ```sh
+    ./yba-ctl.sh createBackup <output_path> [flags]
+    ```
+
+    The `createBackup` command executes the `yb_platform_backup.sh` script to create a backup of your YugabyteDB Anywhere instance. Executing this command requires that you specify the `output_path` where you want the backup `.tar.gz` file to be stored as the first argument to `createBackup`.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:56]:
```suggestion
    The `createBackup` command executes the `yb_platform_backup.sh` script to create a backup of your YugabyteDB Anywhere instance. Specify the `output_path` where you want the backup `.tar.gz` file to be stored.
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+1. Run the `yba-ctl` script using the `restoreBackup` command:
+
+    ```sh
+    yba-ctl restoreBackup <input_path> [flags]
+    ```
+
+    The `restoreBackup` command executes the `yb_platform_backup.sh` to restore from a previously taken backup of your YugabyteDB Anywhere instance. Executing this command requires that you specify the `input_path` to the backup `.tar.gz` file as the only argument to `restoreBackup`.
+
+Upon completion of the preceding steps, the restored YugabyteDB Anywhere is ready to continue managing your universes and clusters.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:76]:
```suggestion
When finished, the restored YugabyteDB Anywhere is ready to continue managing your universes and clusters.
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+{{< /note >}}
+
+## Back up YBA if installed using YBA Installer
+
+If you installed YugabyteDB Anywhere using [YBA installer](../../install-yugabyte-platform/install-software/installer/), perform the following steps:
+
+1. Run the `yba-ctl` script using the `createbackup` command, as follows:
+
+    ```sh
+    ./yba-ctl.sh createBackup <output_path> [flags]
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:53]:
```suggestion
    yba-ctl createBackup <output_path> [flags]
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+{{< note title="Note" >}}
+
+You cannot back up and restore Prometheus metrics data.
+
+{{< /note >}}
+
+## Back up YBA if installed using YBA Installer
+
+If you installed YugabyteDB Anywhere using [YBA installer](../../install-yugabyte-platform/install-software/installer/), perform the following steps:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:48]:
```suggestion
To back up your YugabyteDB Anywhere installation, perform the following steps:
```

**Review by ddhodge — APPROVED** [review]
misc comments

**Comment by chidmuthu** [comment]
I think we should also add some examples of what flags are available (don't need all of them) and highlight they are 1) optional and 2) can be viewed by running createBackup --help

**Comment by aishwarya24** [comment]
> I think we should also add some examples of what flags are available (don't need all of them) and highlight they are 1) optional and 2) can be viewed by running createBackup --help

@chidmuthu - Added the flags description in a table. Can you PTAL?

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+{{< /note >}}
+
+## Back up YBA
+
+To back up your YugabyteDB Anywhere installation, perform the following steps:
+
+1. Run the `createbackup` command, as follows:
+
+    ```sh
+    yba-ctl createBackup <output_path> [flags]
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:53]: Add sudo

**Review by chidmuthu — CHANGES_REQUESTED** [review]
We also need for 2.18, no?

**Comment by aishwarya24** [comment]
> We also need for 2.18, no?

Yes stable is 2.18. So, yes I have it included in stable.

**Review by chidmuthu — APPROVED** [review]

**Pull request merged**